baybe/utils/sampling_algorithms.py

```diff
@@ -70,3 +76,46 @@ def farthest_point_sampling(
         remaining_point_indices.remove(selected_point_index)
 
     return selected_point_indices
+
+
+class SamplingMethod(Enum):
+    """Available sampling methods for dataframes."""
+
+    Random = "Random"
+    """Random Sampling."""
+
+    FPS = "FPS"
+    """Farthest point sampling."""
+
+
+def sample_numerical_df(
+    df: pd.DataFrame, n_points: int, method: SamplingMethod
+) -> pd.DataFrame:
+    """Sample data points from a data frame.
```

Comment:
Perhaps there are also other place where we need to clean it up, but I think we should not say `data frame` in one place and `dataframe` in another. I think in most cases we've used `dataframe` so far